edx_adapt/api/resources/tutor_resources.py

```diff
@@ -1,103 +1,91 @@
-""" This file contains api resources for interacting with the tutor
-as users take the course.
 """
-
-import threading
+This file contains api resources for interacting with the tutor as users take the course.
+"""
 import time
 
-from flask_restful import Resource, abort, reqparse
+from flask_restful import abort, reqparse
 
+from edx_adapt.api.resources.base_resource import BaseResource
 from edx_adapt.data.interface import DataException
 from edx_adapt import logger
 from edx_adapt.select.interface import SelectException
 
-from edx_adapt.misc import psiturk_with_bo
+
+class BaseResources(BaseResource):
```

Comment:
So now there are two `BaseResources(BaseResource)`? This won't make life simpler.